spec/integration/schema_parser_spec.rb

```diff
@@ -10,7 +10,15 @@
             "attributes": {
               "name": "About naming stuff and cache invalidation"
             }
-          }
+          },
+          "included": [
```

Comment:
If you're interested IIRC it turned out that we would first need to add another hash-schema constructor to dry-types based on which we then could define another dry-validation schema which would raise error when unknown keys are passed in.

Here's the fork & branch where I started adding the additional hash-schema contructor. It needed to be a combination of the `symbolized` and the `strict` constuctor. I'm not sure anymore if the `WithDefaults` handling was needed or if it just made things easier.
https://github.com/andreaseger/dry-types/tree/feature/add_strict_with_defaults_symbolized_schema